MekHQ/resources/mekhq/resources/GUI.properties

```
@@ -312,70 +312,115 @@ UnitIconDialog.title=Select Unit Icon
+  \ decision to advance the day.\
+  <br>\
+  <br><i>You can resolve this issue by selecting 'Marketplace' in the taskbar, then 'Astech Pool,'\
+  \ then selecting the option to bring all teams up to full strength. If this does not resolve the\
+  \ issue, you have likely spread your techs too thinly by assigning them to too many units.\
+  \ Consider hiring more techs. Alternatively, you may enable overtime, though this has its own\
+  \ downsides.</i>
+InsufficientMedicsNagDialog.text=%s, our medical teams are short by %d medic%s. This may lead to\
+  \ delays in treatment or personnel remaining untreated. Confirm if you wish to advance the day\
+  \ regardless.\
+  <br>\
+  <br><i>You can resolve this issue by selecting 'Marketplace' in the taskbar, then 'Medic Pool,'\
+  \ then selecting the option to bring all teams up to full strength.</i>
+InvalidFactionNagDialog.text=%s, reports indicate our parent faction is invalid. Possibly\
+  \ destroyed. Confirm your intent to advance the day under these circumstances.\
+  <br>\
+  <br><i>You will need to select a new faction in the campaign options. Failure to do will result\
+  \ in contracts no longer being generated.
+OutstandingScenariosNagDialog.text=%s, our forces are positioned for combat but await your commands to\
+  \ proceed. Alternatively, you may order a tactical withdrawal by advancing the day. Do you wish\
+  \ to proceed without addressing these engagements?\
+  <br>\
+  <br>The following scenarios await resolution:%s%s
+OutstandingScenariosNagDialog.stratCon=<br>\
+  <br><i>We can pay 1 CVP to tactically withdraw from any scenario marked as 'Critical.' Withdrawal is\
+  \ free for all other scenarios.</i>
+PregnantCombatantNagDialog.text=%s, pregnant personnel have been identified in our TO&E, putting\
+  \ them at risk of harm. Reassignment is advised. Do you wish to advance the day without\
+  \ addressing this concern?
+DeploymentShortfallNagDialog.text=%s, deployment levels fall below requirements for one or more active\
+  \ contracts. While advancing the day is an option, it may strain relations with our employer. Do\
+  \ you wish to continue without addressing this?\
+  <br>\
+  <br><i>You should head to the Briefing Room and address this shortfall. Remember, Combat Teams\
+  \ assigned to the Auxiliary or Reserve roles do not count towards deployment levels. Nor do\
+  \ Combat Teams assigned to the Training role, unless the contract is Cadre Duty.</i>
+PrisonersNagDialog.text=%s, our forces are still holding prisoners of war. It is important to\
+  \ consider their status before advancing the day. Do you wish to continue without resolving this?\
+  <br>\
+  <br><i>You should head to the Personnel tab and set the filter to 'Prisoners'. Right-clicking on\
+  \ a prisoner will allow you to free, ransom, or execute them. If you are currently off-world, you\
+  \ can choose to jettison your prisoners</i>
```

```suggestion
  \ can choose to jettison your prisoners.</i>
```

May also consider adding a :( to the end of the sentence, lol.